sdv/data_processing/data_processor.py

```diff
@@ -103,6 +104,104 @@ def __init__(self, metadata, enforce_rounding=True, enforce_min_max_values=True,
         if self._primary_key:
             self._keys.append(self._primary_key)
 
+    def _get_column_in_multi_column_transformer(self):
+        """Get the columns that are part of a multi column transformer.
+
+        Returns:
+            list:
+                A list of columns that are part of a multi column transformer.
+        """
+        return [
+            col for col_tuple in self.columns_to_multi_col_transformer for col in col_tuple
+        ]
+
+    def _import_address_transformers(self):
+        """Try to import the address transformers."""
+        try:
+            from rdt.transformers import RandomLocationGenerator, RegionalAnonymizer
+            self.RandomLocationGenerator = RandomLocationGenerator
+            self.RegionalAnonymizer = RegionalAnonymizer
+        except ImportError:
+            raise ImportError(
+                'You must have SDV Enterprise with the address add-on to use the address features'
+            )
```

Comment:
This should be renamed to `check_address_transformers_present` or something like that.
Basically you should ensure that you have access to the `RandomLocationGenerator` which should (I guess) be in `rdt.transformers`. Therefore you should be able to access it trough `rdt.transformers.RandomLocationGenerator`.
On that case you can check like this:
```

In [1]: import rdt
    
In [2]: hasattr(rdt.transformers, 'FloatFormatter')
Out[2]: True

In [3]: 
```
Then the logic is quiet simple:
```python
if not hasattr(rdt.transformers, 'RandomLocationGenerator'):
    raise AttributeError('You must have SDV-Enterprsie ...')
```